### PR TITLE
clean api server cruft

### PR DESCRIPTION
**Pull request by deads2k**
Some cruft has developed over refactors.  Remove that cruft.

@liggitt probably last in the chain so far

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/33842)

### CODE REVIEW

**Comment by deads2k** [comment]
@ncdc cleaning up some dead fields before fixing up legacy api paths.

**Comment by ncdc** [comment]
@deads2k build failures in integration tests?

**Comment by deads2k** [comment]
@ncdc fixed.  Eliminated nonymous inclusion and their integration tests wire deeply

File: pkg/master/master.go [hunk 1]
```diff
@@ -253,12 +253,37 @@ func (c completedConfig) New() (*Master, error) {
...
+
+	m.InstallGeneralEndpoints(c.Config)
 
 	return m, nil
 }
 
-func (m *Master) InstallAPIs(c *Config) {
+// TODO this needs to be refactored so we have a way to add general health checks to genericapiserver
+// TODO profiling should be generic
+func (m *Master) InstallGeneralEndpoints(c *Config) {
```
Comment (ncdc) [inline pkg/master/master.go]: I don't really like the name of this function. Do you have any better suggestions?

**Review by ncdc — APPROVED** [review]
1 minor comment (non blocking). lgtm

File: pkg/master/master.go (hunk 1 above)
Comment (deads2k) [inline pkg/master/master.go]: > I don't really like the name of this function. Do you have any better suggestions?

I see us completely refactoring `healthz` to allow composable checks.  Two weeks and its gone.

**Comment by deads2k** [comment]
@k8s-bot gce e2e test this

**Comment by deads2k** [comment]
@mml another catastrophic failure.  This time `Up`. https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/33842/kubernetes-pull-build-test-e2e-gce/62660/

**Comment by deads2k** [comment]
@k8s-bot cvm gce e2e test this

**Pull request merged**